src/components/DocumentationTopic.vue

```diff
@@ -30,6 +30,14 @@
           <slot name="above-hero-content" />
         </template>
         <slot name="above-title" />
+        <Hierarchy
+          v-if="hierarchyItems.length && !enableMinimized"
```

Comment:
I think we discussed hiding Hierarchy in IDE mode as well, did that change?

Reply:
We could make this change in the future if we want. For now, I think it's ok that this component is conditionally rendered as long as the data is provided.